cmd/sharded-test-server/frontproxy.go

```diff
@@ -193,7 +193,7 @@ func startFrontProxy(
 
 		// intentionally load again every iteration because it can change
 		configLoader := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(&clientcmd.ClientConfigLoadingRules{ExplicitPath: filepath.Join(workDirPath, ".kcp/admin.kubeconfig")},
-			&clientcmd.ConfigOverrides{CurrentContext: "system:admin"},
+			&clientcmd.ConfigOverrides{CurrentContext: "base"},
```

Comment:
@ncdc I can see this looks correct looking at the kubeconfig, but I don't fully understand why this is now required (e.g why did it work before), was this always broken and we just didn't see it because we didn't wait for the shard readyz?

Anyway, I do see that without this change the shard never becomes ready so clearly it's needed now :)

Reply:
Yes

Reply:
ah, looks like `system:admin` is an invalid context